main.go

```diff
@@ -84,37 +86,64 @@ func mustExec(sqlStmt string) {
 	}
 }
 
+func query(sqlStmt string) (*sql.Rows, error) {
+	db := connPool.Get().(*sql.DB)
+	defer connPool.Put(db)
+	log.Debug("query sql:", sqlStmt)
+	rows, err := db.Query(sqlStmt)
+	return rows, err
+}
+func checkQuery(sqlStmt string, expectRows int) {
+	rows, err := query(sqlStmt)
+	if err != nil {
+		log.Error(sqlStmt)
+	}
+	actGot := 0
+	for rows.Next() {
+		actGot++
+		if actGot > expectRows {
+			log.Fatal(fmt.Sprintf("sql return count does not match. actual got %d great than expect %d\n", actGot, expectRows))
+			break
+		}
+	}
+	if actGot < expectRows {
+		log.Fatal(fmt.Sprintf("sql return count does not match. actual got %d less than expect %d\n", actGot, expectRows))
+	}
+}
+
 func timing(desc string, fn func()) {
 	fmt.Printf("%s ... [START]\n", desc)
 	c := time.Now()
 	fn()
-	fmt.Printf("%s ... elapse : %v [DONE]\n", desc, time.Since(c))
+	fmt.Printf("%s ... elapse : %fs [DONE]\n", desc, time.Since(c).Seconds())
 }
 
+func doInsertTestData(workerId int, wg *sync.WaitGroup, idChan chan int) {
+	defer wg.Done()
+	for {
+		id, ok := <-idChan
+		// All data are sent
+		if !ok {
+			return
+		}
+		var bulks []string
+		for i := 0; i < *nCols; i++ {
+			// Fill dummy data, and generate SQL
+			buf := bytes.Repeat([]byte{'A'}, *bulkSize)
+			bulks = append(bulks, fmt.Sprintf("\"%s\"", string(buf)))
+		}
+		sql := fmt.Sprintf("INSERT INTO %s VALUES(%d, %s);", tableName, id, strings.Join(bulks, ","))
+		mustExec(sql)
+	}
+}
 func insertTestData(rows int, workers int) error {
+	createTable(ForceDrop)
```

Comment:
Use lowercase.
 You should never use uppercase except it's an exported function/variable.